amp/WEB-INF/src/org/digijava/module/aim/util/ActivityVersionUtil.java

```diff
@@ -327,4 +369,389 @@ public static AmpActivityVersion initializeActivity(AmpActivityVersion act) thro
         }
         return act;
     }
+    
+    
+
+    
+    public static Map<String, List<CompareOutput>> compareActivities(Long activityOneId,Long activityTwoId ) throws Exception{ 
```

Comment:
please format compareActivites *only*. Please do not commit if by mistake the IDE formats anything but compareActivities method. Since this method is entirely new in this class is the opportunity to format it